lib/money/bank/wego_money_bank.rb

```diff
@@ -47,15 +47,17 @@ def update_rates
       end
 
       def fetch_from_url
-        open(url).read
+        open(url, read_timeout: 3).read
+      rescue OpenURI::HTTPError
+        nil
```

Comment:
@tamnguyenit @codenut @kates @viettienn 

any advise what to do here? if this keeps on failing we have no way of knowing. the rates will be out of date.